src/i18n/messages/ar.json

```diff
@@ -1,2 +1,73 @@
+  "gradebook.BulkManagementHistoryView": "Below is a log of previous grade imports.  To download a CSV of your gradebook and import grades for override, return to the Gradebook.  Please note, after importing grades, it may take a few seconds to process the override.",
+  "gradebook.BulkManagementHistoryView.successDialog": "CSV processing. File uploads may take several minutes to complete.",
+  "gradebook.GradebookFilters.assignmentsFilterLabel": "Assignments",
+  "gradebook.GradebookFilters.overallGradeFilterLabel": "Overall Grade",
+  "gradebook.GradebookFilters.studentGroupsFilterLabel": "Student Groups",
+  "gradebook.GradebookFilters.includeCourseTeamMembersFilterLabel": "Include Course Team Members",
+  "gradebook.GradebookFilters.assignmentFilterLabel": "Assignment",
+  "gradebook.GradebookFilters.assignmentTypesLabel": "Assignment Types",
+  "gradebook.GradebookFilters.maxGradeFilterLabel": "Max Grade",
+  "gradebook.GradebookFilters.minGradeFilterLabel": "Min Grade",
+  "gradebook.GradebookFilters.cohorts": "Cohorts",
+  "gradebook.GradebookFilters.cohortsAll": "Cohort-All",
+  "gradebook.GradebookFilters.tracks": "Tracks",
+  "gradebook.GradebookFilters.trackAll": "Track-All",
+  "gradebook.GradebookFilters.closeFilters": "Close Filters",
+  "gradebook.GradebookHeader.backButton": "Back to Dashboard",
+  "gradebook.GradebookHeader.appLabel": "Gradebook",
+  "gradebook.GradebookHeader.frozenWarning": "The grades for this course are now frozen. Editing of grades is no longer allowed.",
+  "gradebook.GradebookHeader.unauthorizedWarning": "You are not authorized to view the gradebook for this course.",
+  "gradebook.GradebookHeader.toActivityLogButton": "View Bulk Management History",
+  "gradebook.GradebookHeader.toGradesView": "Return to Gradebook",
+  "gradebook.GradesView.BulkManagementControls.bulkManagementLabel": "Download Grades",
+  "gradebook.GradesView.EditModal.headers.assignment": "Assignment",
+  "gradebook.GradesView.EditModal.headers.currentGrade": "Current Grade",
+  "gradebook.GradesView.EditModal.headers.originalGrade": "Original Grade",
+  "gradebook.GradesView.EditModal.headers.student": "Student",
+  "gradebook.GradesView.EditModal.title": "Edit Grades",
+  "gradebook.GradesView.EditModal.closeText": "Cancel",
+  "gradebook.GradesView.EditModal.contactSupport": "Showing most recent actions (max 5). To see more, please contact support",
+  "gradebook.GradesView.EditModal.saveVisibility": "Note: Once you save, your changes will be visible to students.",
+  "gradebook.GradesView.EditModal.saveGrade": "Save Grades",
+  "gradebook.GradesView.EditModal.Overrides.adjustedGradeHeader": "Adjusted grade",
+  "gradebook.GradesView.EditModal.Overrides.dateHeader": "Date",
+  "gradebook.GradesView.EditModal.Overrides.graderHeader": "Grader",
+  "gradebook.GradesView.EditModal.Overrides.reasonHeader": "Reason",
+  "gradebook.GradesTab.usersVisibilityLabel'": "Showing {filteredUsers} of {totalUsers} total learners",
```

Comment:
I know this PR is not focused on the actual string definitions, but this one is particularly egregious. `filteredUsers` as a variable implies a list of users, but contextually I think it must be a number. `Showing {numFilteredUsers} of {numTotalUsers}` would be significantly clearer. Flagging so team engineers can understand.